keras_cv/layers/preprocessing/equalize_test.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing.equalize import Equalize
+
+
+class EqualizeTest(tf.test.TestCase):
```

Comment:
We should consider using `layer_test` from Keras to cover standard layer features (including serialization)

Reply:
On this note, we could export `layer_test` as an `keras.__internal__` API so as to guarantee API stability

Reply:
Yea, I will send a cl for that. addon was using layer_test as well.

Reply:
Sending https://github.com/keras-team/keras/pull/16021.